bag_transfer/migrations/0056_add_bagit_profile_values.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2.16 on 2025-02-27 21:52
+
+from django.db import migrations
+
+# Add default technical values to existing BagIt Profiles
+def add_bagit_profile_values(apps, schema_editor):
+    BagItProfile = apps.get_model('bag_transfer', 'BagItProfile')
+    ManifestsAllowed = apps.get_model('bag_transfer', 'ManifestsAllowed')
+    AcceptSerialization = apps.get_model('bag_transfer', 'AcceptSerialization')
+    AcceptBagItVersion = apps.get_model('bag_transfer', 'AcceptBagItVersion')
+
+    sha256 = ManifestsAllowed.objects.get(name="sha256")
+    sha512 = ManifestsAllowed.objects.get(name="sha512")
+    zip_format = AcceptSerialization.objects.get(name="application/zip")
+    tar_format = AcceptSerialization.objects.get(name="application/x-tar")
+    gzip_format = AcceptSerialization.objects.get(name="application/x-gzip")
+    bagit_version_097 = AcceptBagItVersion.objects.get(name="0.97")
+
+    # Loop through existing BagIt Profiles and update them
+    for profile in BagItProfile.objects.all():
+        profile.manifests_allowed.set([sha256, sha512])
+        profile.accept_serialization.set([zip_format, tar_format, gzip_format])
+        profile.accept_bagit_version.set([bagit_version_097])
```

Comment:
I wonder if we should set this to additional versions, or not set it at all. It feels like something that is pretty invisible to most folks submitting packages, and/or not really in their control because it's based on what tool they're using.